README.rst

```diff
@@ -1,130 +1,117 @@
-generically applicable algorithms.
+============
+Installation
+============
 
-To see what coala can do for you and your language, take a look at
-`our capabilities listing <https://github.com/coala-analyzer/bear-docs/blob/master/README.rst>`__.
+To install the **latest stable version** run:
 
-*coala* is written with a lower case "c".
+.. code-block:: bash
 
-Read more at our `documentation <http://coala.rtfd.org/>`__.
+    $ pip3 install coala-bears
 
-Why use coala?
---------------
+To install the latest development version run:
 
-- *coala* provides **built-in checking routines** (named bears in *coala*).
-- **Serves your requirement**: You can easily write your own checks (using
-  bears).
-- *coala* provides **user-friendly interfaces** like json, formatted and
-  interactive output in the CLI and *plugins for various editors* are
-  available as well.
-- **Optimal performance**: *coala* manages parallelizing the checking-routines
-  without you having to worry.
-- **Unified interface**: One tool for all programming languages.
+.. code-block:: bash
 
-If you want to learn more about *coala*, its functionality and its usage,
-please take a look at our
-`tutorial <http://coala.rtfd.org/en/latest/Users/Tutorials/Tutorial.html>`__.
+    $ pip3 install coala-bears --pre
 
-Installation
-------------
+|PyPI| |Windows| |Linux|
 
-To install *coala* with the official set of analysis routines you can simply run
-``pip3 install coala-bears``.
+-----
 
-*coala* only, without the official bears, can be installed with
-``pip3 install coala``. If you need more information about the installation and
-dependencies, take a look at our `installation documentation
-<http://coala.rtfd.org/en/latest/Users/Install.html>`__.
+=====
+Usage
+=====
 
-The latest code from master is automatically deployed to PyPI as a
-development version. Get it with ``pip3 install coala --pre``.
+To get the list of arguments and their meaning, run:
 
-|PyPI|
+.. code-block:: bash
 
-Authors
--------
+    $ coala --help
 
-*coala* is maintained by a growing community. Please take a look at the
-meta information in `setup.py <setup.py>`__ for current maintainers.
+-----
 
-Project Status
---------------
+=======
+Support
+=======
 
-|Linux Build Status| |Windows Build status|
+Feel free to contact us at our `Gitter page <https://gitter.im/coala-analyzer/coala>`__, we'd be happy to help!
 
-|Scrutinizer Code Quality| |codecov.io|
+|gitter|
 
-|Documentation Status| |Gitmate|
+-----
 
-Newcomers Guide and Getting Involved
-------------------------------------
+================
+Getting Involved
+================
 
-If you are new and would like to contribute, read our `Getting Involved Site
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__!
+If you would like to contribute, please take a look at our `Getting Involved <http://coala.readthedocs.io/en/latest/Getting_Involved/README.html>`__ page or ask us at our gitter channel. We appreciate any help!
```

Comment:
in this section, also mention something like: ``We also prove a newcomer guide, which helps you get started by fixing an issue alone: guide link`` 
http://coala.readthedocs.io/en/latest/Getting_Involved/Newcomers.html